clusters/app.ci/pr-reminder/pr-reminder-config.yaml

```diff
@@ -71,3 +71,40 @@ data:
         repos:
           - openshift/baremetal-runtimecfg
           - openshift/kubernetes-nmstate
+      - teamMembers:
+          - fbertina
+          - hekumar
+          - jdobson
```

Comment:
```suggestion
          - dobsonj
```

Reply:
Are these github handles or slack handles?
If github: Roman, Hemant, and Fabio have different handle names.

Reply:
The [guide](https://docs.ci.openshift.org/docs/how-tos/pr-reminder/#how-to-configure-it-for-your-team) says it should be kerberos id. And according to rover your id should be `jdobson` which is also the primary email address, I suppose the other one is just an alias?

So I think it's correct how it is, or not?

Reply:
Yes, it looks correct to me.
(And we can fix it later if it's not)